uk.co.compucorp.civicrm.tasksassignments/js/src/tasks-assignments/filters/unique.filter.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-env amd */
+
+define([
+  'common/lodash',
+], function (_) {
+  'use strict';
+
+  uniqueFilter.__name = 'unique';
+  uniqueFilter.$inject = ['$filter', '$rootScope', '$log'];
+
+  function uniqueFilter ($filter, $rootScope, $log) {
+    $log.debug('Filter: unique');
+
+    /**
+     * Filters the array to return unique values
+     *
+     * @param  {Array} inputArray
+     * @param  {Array} identifier to filter by
```

Comment:
Doesn't uniq accept only one param? https://lodash.com/docs/4.17.4#uniq

Reply:
We are using Lodash 3.10.1 https://lodash.com/docs/3.10.1#uniq. It can accept second param as well incase of list of objects.

Reply:
@bmnepali Thanks, I see now. That means when we update lodash we have to be sure our tests cover this. Otherwise we will have instant regressions and they may not be easy to spot. May I ask you to check where else _.uniq is used in the extension with 2 params?

In lodash 4 _.uniq accepts 1 param, for additional identifier you use _.uniqBy.

Reply:
@igorpavlov 
> It also can return false positives like _.uniq(_.map(allEntitlements, function (entitlement) { but anyway, that is for a single check. Hope it helps!

Here we are using `_.uniq()` only to find the unique `ids`, so i don't think, there are chance of getting false positive values. 
```js
var authorsIDs = _.uniq(_.map(allEntitlements, function (entitlement) {
    return entitlement.comment_author_id;
}));
```